.github/ISSUE_TEMPLATE/comp_extensions_enrichment_AspNetCore.md

```diff
@@ -0,0 +1,54 @@
+---
+name: OpenTelemetry.Extensions.Enrichment.AspNetCore
+about: Issue with OpenTelemetry.Extensions.Enrichment.AspNetCore
+labels: comp:extensions.enrichment.AspNetCore
```

Comment:
```suggestion
labels: comp:extensions.enrichment.aspnetcore
```

Reply:
Thanks. I was confused, for example, [here ](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/.github/ISSUE_TEMPLATE/comp_instrumentation_AspNet.TelemetryHttpModule.md) and [here ](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/.github/ISSUE_TEMPLATE/comp_instrumentation_AspNet.md )upper case letters are used in labels

Reply:
Fixed